packages/Input/tests/input.spec.js

```diff
@@ -1,13 +1,51 @@
 import React from "react";
-import { render } from "@testing-library/react";
+import { render, fireEvent } from "@testing-library/react";
 import { ShirtSizes } from "@paprika/helpers/lib/customPropTypes";
 import Input from "../src";
 
+function renderComponent(props = {}) {
+  const rendered = render(<Input {...props} />);
+  const { rerender } = rendered;
+
+  return {
+    ...rendered,
+    rerender: updatedProps => {
+      rerender(<Input {...updatedProps} />);
+    },
+  };
+}
+
 describe("Input", () => {
-  it("should be visible", () => {
-    const { container } = render(<Input onChange={() => {}} placeholder="First Name" size={ShirtSizes.LARGE} />);
+  it("should be visible and render large text", () => {
+    const { container } = renderComponent({ size: ShirtSizes.LARGE });
     expect(container.firstChild.classList.contains("form-input--large")).toBe(true);
   });
-});
 
-// TODO: add more tests
+  it("should change value when value changes", async () => {
+    let changeValue = null;
```

Comment:
Can you make variables for `init value` and `input change`, and use those throughout? May make it easier to follow:

    const initialValue = 'init value';
    const updatedValue = 'input change';
    ...